<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>958</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="idtext1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="idtext2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadbutton1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadbutton2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copybutton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>copy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextbutton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic1_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathtext">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>del</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic1_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic1_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic2_1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic2_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>640</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic2_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="openpath"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <action name="openpath">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>copybutton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>copyfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextbutton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>nextdir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>-18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadbutton1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>id1loadOnclick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadbutton2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>id2loadOnclick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>622</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>id1delOnclick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>id2delOnclick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>copyfile()</slot>
  <slot>nextdir()</slot>
  <slot>id1loadOnclick()</slot>
  <slot>id2loadOnclick()</slot>
  <slot>id1delOnclick()</slot>
  <slot>id2delOnclick()</slot>
 </slots>
</ui>
